src/components/lib/VDropdown/VDropdown.vue

```diff
@@ -145,7 +145,7 @@
 }
 
 .custom-select.disabled .custom-select__trigger {
-  cursor: grab;
+  cursor: default;
```

Comment:
If the intent is to show the dropdown is clickable, we should use `cursor: pointer` here.

```suggestion
  cursor: pointer;
```

Reply:
Shouldn't we rather use `context-menu` ? 

![image](https://user-images.githubusercontent.com/29047033/147723121-d1639822-8457-4bce-ace5-ec013fa206c9.png)

Reply:
Never used `context-menu` before or see it being used in the wild.

I believe `pointer` is more common and better understood by users.

Please go ahead and use what you like. 👍🏼

Reply:
I agree with Ajith here, I have not seen websites using context menu pointer. I saw it last time on windows 98 :D

Reply:
Actually this is for the disabled state. For active state we have `cursor: pointer` for VDropdown component. 
This is not a native select element as styling it was limited or atleast I didn't find a way to style option elements the way we wanted it to look, so I had created a custom dropdown element using divs and spans. I didn't knew how to attach disabled psuedoselector to div so I just used class.

So in this case when dropdown is disabled `cursor: default` or `cursor: not-allowed`. 

And for `cursor: grab`, maybe while I was creating the component I had it inserted accidentally and since chrome didn't show grab cursor, I also didn't realize that mistake while testing, but it does shows in other browsers like firefox